src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2105 @@
+        /// <exception cref="OutOfMemoryException">This exception can be due to out of memory.</exception>
+        /// <pre>
+        /// The State must be Ready.
+        /// </pre>
+        public static void DoAction(SendEventType type, string sendEvent)
+        {
+            ErrorCode error = VcMgrDoAction(type, sendEvent);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "DoAction Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Sends the specific engine request to the vc-service.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="engineAppId">A specific engine's app id</param>
+        /// <param name="evt">A engine service user request event</param>
+        /// <param name="request">A engine service user request text</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
+        /// <exception cref="NotSupportedException">This exception can be due to not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">This exception can be due to permission denied.</exception>
+        public static void SendSpecificEngineRequest(string engineAppId, string evt, string request)
+        {
+            ErrorCode error = VcMgrSendSpecificEngineRequest(engineAppId, evt, request);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "SendSpecificEngineRequest Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Starts recognition.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="exclusiveCommandOption">Exclusive command option</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
```

Comment:
If this exception can be thrown in both cases, please write it as a single instance of `<exception cref="InvalidOperationException">`.